scipy/signal/_peak_finding_utils.pyx

```diff
@@ -283,6 +295,11 @@ def _peak_widths(np.float64_t[::1] x not None,
         ``0 <= left_base <= peak <= right_base < x.shape[0]`` for each peak or
         if `peaks`, `left_bases` and `right_bases` don't share the same shape.
 
+    Warnings
```

Comment:
Is this maybe maint to be `Raises`? I didn't check but feels like it.

Reply:
See [comment](https://github.com/scipy/scipy/pull/9126#discussion_r209457750) above on [numpydoc's](https://numpydoc.readthedocs.io/en/latest/format.html#sections) section `Warnings`.

Reply:
Sure, no problem.